__init__.py

```diff
@@ -428,7 +430,11 @@ def show_coverage_report(bv):
         function_addr = function_dict[mangled_name].start
         report += "  0x%08x  %s%s : %.2f%% coverage\t( %-3d / %3d blocks)\n" % \
                   (function_addr, name, pad, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
-    show_plain_text_report(title, report)
+        report_html += "<tr><td><a href='binaryninja:?expr=0x%x'>0x%08x</a></td><td>%s</td><td>%.2f%% coverage</td><td>%-3d / %3d blocks</td></tr>\n" % \
+                  (function_addr, function_addr, name, pad, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
```

Comment:
```suggestion
                  (function_addr, function_addr, name, stats.coverage_percent, stats.blocks_covered, stats.blocks_total)
```

You would either need to make `%s` into `%s%s` in the previous line, or remove this. The table should format for you regardless.

Reply:
Thanks, this was my intention. As I said, I didn't actually test it. 😉